### PR TITLE
Webhook openeye to docker image

### PR DESCRIPTION
**Pull request by ttayson**
Added a Docker setup option to the end of the README

### CODE REVIEW

File: webhooks/Webhook_OpenEye/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM python:3.9.7
```
Comment (marcbelmont) [inline webhooks/Webhook_OpenEye/Dockerfile:1]: can you switch to 3.9-slim?

**Review by marcbelmont — COMMENTED** [review]
- why did you remove docker/WSL_Install.bat?

**Comment by ttayson** [comment]
> * por que você removeu docker/WSL_Install.bat?

Sorry @marcbelmont , it was an old commit that was added wrong.

File: webhooks/Webhook_OpenEye/Dockerfile (hunk 1 above)
Comment (ttayson) [reply webhooks/Webhook_OpenEye/Dockerfile:1]: @marcbelmont i will do it

**Review by marcbelmont — APPROVED** [review]

File: webhooks/Webhook_OpenEye/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+   ```bash
+   docker build --tag platerecognizer/webhook-openeye .
+
+   ```
+
+2. Run the image
+
+   ```bash
+   docker run --rm -t -p 5000:5000 -e AKI_TOKEN=XXXXXXXXXXXX -e AKS_TOKEN=YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY platerecognizer/webhook-openeye
```
Comment (marcbelmont) [inline webhooks/Webhook_OpenEye/README.md:85]: Can you confirm that you tested this and that it worked?

**Comment by marcbelmont** [comment]
do you want me to add that to docker hub?

**Pull request merged**